alarms/Boxcar/boxcar_alarm.py

```diff
@@ -17,7 +17,7 @@ class Boxcar_Alarm(Alarm):
 	_defaults = {
 		'pokemon':{
 			'title':"A wild <pkmn> has appeared!",
-			'long_message':"Available <a href='<gmaps>'>here</a> until <24h_time> (<time_left>).",
```

Comment:
I think it needs to be nonstandard because:
1. We primarily support PokemonGo-Map.
2. PokemonGo-Map doesn't support this feature.

I understand the way you have it now replaces it with empty space, but it should function similar to the other dynamic text options and instead leave a 'None' if the information was not provided in the POST request. If you are using a scanner with this function, it makes sense that you need to let PokeAlarm know that you want to see it. Otherwise it should be ignored.